src/fragments/lib/auth/ios/signout/10_local_signout.mdx

```diff
@@ -23,14 +34,33 @@ func signOutLocally() async {
 func signOutLocally() -> AnyCancellable {
     Amplify.Publisher.create {
         await Amplify.Auth.signOut()
-    }.sink {
-        if case let .failure(authError) = $0 {
-            print("Sign out failed with error \(authError)")
+    }.sink(
+        receiveCompletion: {
+            if case let .failure(authError) = $0 {
+                print("Sign out failed with error \(authError)")
+            }
+        },
+        receiveValue: { result in
+            if let signOutResult = result as? AWSCognitoSignOutResult {
+                print("Local signout successful: \(signOutResult.signedOutLocally)")
+                switch signOutResult {
```

Comment:
nit: should we use guard instead? for the Combine example, it's getting a few indentations deep. if we do, then should use it for the Async/Await example as well

Reply:
good call. updated in [3cf480a](https://github.com/aws-amplify/docs/pull/4670/commits/3cf480a4e551f4d0aec6a611934e8e850c8fd07c)